product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx

```diff
@@ -11,48 +11,48 @@ You must configure the PEM database server to work with PgBouncer. This example
 
 1.  Create a dedicated user named pgbouncer on the PEM database server:
 
-    ```ini
-    pem=# CREATE USER pgbouncer PASSWORD 'ANY_PASSWORD' LOGIN;
+    ```sql
+    CREATE USER pgbouncer PASSWORD 'ANY_PASSWORD' LOGIN;
     CREATE ROLE
     ```
 
 2.  Create a user named pem_admin1 (not a superuser) with `pem_admin` and `pem_agent_pool role` membership on the PEM database server:
 
-    ```ini
-    pem=# CREATE USER pem_admin1 PASSWORD 'ANY_PASSWORD' LOGIN
+    ```sql
+    CREATE USER pem_admin1 PASSWORD 'ANY_PASSWORD' LOGIN
     CREATEROLE;
```

Comment:
CREATEROLE looks like it should be on the same line as the command - let the code block wrapper take the strain.

Reply:
got it 👍